client/my-sites/customize/main.jsx

```diff
@@ -326,6 +331,18 @@ var Customize = React.createClass( {
 } );
 
 export default connect(
-	( state, props ) => props,
+	( state, props ) => {
+		const siteId = props.sites.getSite( props.domain ).ID;
```

Comment:
This isn't a real selector -- can we use `getSelectedSiteId` from `state/ui/selectors` instead?